kythe/cxx/tools/BUILD

```diff
@@ -18,7 +18,6 @@ cc_library(
         "//kythe/proto:analysis_cc_proto",
         "//kythe/proto:buildinfo_cc_proto",
         "//kythe/proto:cxx_cc_proto",
-        "//kythe/proto:filecontext_cc_proto",
```

Comment:
Oops. This was a mistake, I did not mean to remove this. (@schroederc fixed it in #3571)